api/pom.xml

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why remove these files?

Reply:
I think this project should have few dependencies. The original project has too many dependencies.
I did some refactoring.
In the root directory pom.xml It's about the dependencies you need.
Without the test dependencies, now you just need the following dependencies.
 
```base
       <dependency>
            <groupId>redis.clients</groupId>
            <artifactId>jedis</artifactId>
            <version>3.1.0</version>
        </dependency>
        <dependency>
            <groupId>com.distkv</groupId>
            <artifactId>distkv-client</artifactId>
            <version>0.1.4-SNAPSHOT</version>
        </dependency>
```